### PR TITLE
feat: truncate did and fix style issue

### LINKED ISSUE #88: DID showing incorrectly
In version `@verida/vue-account": "^0.1.38"` the DID is showing up incorrectly:

image

(as seen on https://explorer.verida.network/).

It should show `did:vda:testnet:0x69...14d` or similar.

If that is too long, perhaps `did:vda:testnet:...14d` would work?

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- Added functionality to truncate did from `did:vda:testnet:0x2b160b311106de4a899B5AD3D934ADD9c8516Add` to  `did:vda:0x2449..`
- Fix CSS styles issue 

closes #88

### CODE REVIEW

File: components/account/src/lib-components/vda-account.vue [hunk 1]
```diff
@@ -141,6 +137,27 @@ export default /*#__PURE__*/ defineComponent({
     },
   },
 
+  computed: {
+    /** method to remove the "testnet" part in the did and trucate into e.g "did:vda:0x2449.."  format */
```
Comment (aurelticot) [inline components/account/src/lib-components/vda-account.vue:141]: My understanding was the purpose was not to hide testnet from the DID method.

Using `split(":")` is ok but the goal is to extract the whole did method (several elements separated by ":") from actual key (the last item from the split array).
Is then to truncate the key only.

File: components/account/src/lib-components/vda-account.vue [hunk 2]
```diff
@@ -157,11 +174,8 @@ export default /*#__PURE__*/ defineComponent({
 
   methods: {
     copyToClipBoard(value: string) {
-      // @ts-ignore
-      this.$copyText(value);
-
+      navigator.clipboard?.writeText(value);
       this.isCopied = true;
```
Comment (aurelticot) [inline components/account/src/lib-components/vda-account.vue:178]: `navigator.clipboard` can be undefined, so the value may not be copied.

**Review by aurelticot — CHANGES_REQUESTED** [review]

File: components/account/src/lib-components/vda-account.vue (hunk 1 above)
Comment (cmcWebCode40) [reply components/account/src/lib-components/vda-account.vue:141]: do you mean I should truncate only the last item from the split array ?

eg. `did:vda:testnet:0xC64...` ?

Comment (aurelticot) [reply components/account/src/lib-components/vda-account.vue:141]: Yes

File: components/account/src/lib-components/vda-account.vue (hunk 2 above)
Comment (cmcWebCode40) [reply components/account/src/lib-components/vda-account.vue:178]: would you suggest I revert to using the previous copyToClipoboard library or handle when its undefined ?

Comment (aurelticot) [reply components/account/src/lib-components/vda-account.vue:178]: I think it's a feature we don't need a library for, but then, our code must be robust implementing it.

The main question is, should you handle it in this unrelated PR or in a dedicated one?
(And should you have started it in this unrelated PR in the first place?)

Comment (cmcWebCode40) [reply components/account/src/lib-components/vda-account.vue:178]: ok, i think I will have to open a different ticket to implement that feature

File: components/account/src/lib-components/vda-account.vue (hunk 1 above)
Comment (cmcWebCode40) [reply components/account/src/lib-components/vda-account.vue]: fixed

File: components/account/src/lib-components/vda-account.vue (hunk 2 above)
Comment (cmcWebCode40) [reply components/account/src/lib-components/vda-account.vue:174]: changes revert will fixed on its ticket

File: components/account/src/lib-components/vda-login.vue [hunk 3]
```diff
@@ -173,43 +169,43 @@ export default /*#__PURE__*/ defineComponent({
...
   border-radius: 100%;
 
   display: inline-block;
   width: 10px;
   height: 10px;
   margin: 3px 2px;
-  -webkit-animation: loader1 0.8s linear infinite alternate;
-  animation: loader1 0.8s linear infinite alternate;
+  -webkit-animation: loader-icon 0.8s linear infinite alternate;
+  animation: loader-icon 0.8s linear infinite alternate;
```
[the remark this answers is not in the document]
Comment (cmcWebCode40) [reply components/account/src/lib-components/vda-login.vue:186]: fixed

**Review by aurelticot — APPROVED** [review]

**Pull request merged**